### PR TITLE
Prepare for release v1.9.0

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,612 @@
...
+* Added support for non-k8s nodes to register to a k8s cluster via new option `--join-cluster` (#13309, @jrajahalme)
+* build: Experimental multi-platform images (#12013, @errordeveloper)
+* change default docker image repository from docker.io to quay.io (Backport PR #13957, Upstream PR #13937, @aanm)
+* cilium: switch to mq/fq and add bandwidth manager (#12868, @borkmann)
+* Cilium operator HA mode (#12409, @fristonio)
+* Direct routing performance improvement through new tc/BPF-only based host forwarding mode w/o passing to upper stack. (#13330, @borkmann)
+* Helm charts have been fully re-structured to a single chart for cilium with no dependency on sub-charts. More than 170 global values have been properly scoped to the cilium chart with sane defaults defined. Users upgrading from prior versions of cilium should be sure to read the upgrade guide for specific instructions. (#13259, @seanmwinn)
+* Implement eBPF native redirect functionality (#12831, @aditighag)
+* Implement proxy redirection logic in eBPF (#11279, @joestringer)
+* Support for external workloads (e.g., VMs) has been added (beta) (Backport PR #13964, Upstream PR #13940, @jrajahalme)
```
Comment (pchaigno) [inline CHANGELOG.md:22]: This line seems redundant with line 14. #13940 refers to the GSG, so maybe not a major change on its own? Maybe we can just remove this line from the 1.9.0 changelogs.

**Review by pchaigno — APPROVED** [review]
Minor comment on the major changelogs.

**Review by fristonio — APPROVED** [review]
LGTM :rocket: :tada:

File: CHANGELOG.md (hunk 1 above)
Comment (aanm) [reply CHANGELOG.md:22]: I've removed line 14

**Pull request merged**